projects/core/src/util/converter.service.ts

```diff
@@ -24,8 +26,20 @@ export interface Converter<S, T> {
 @Injectable({
   providedIn: 'root',
 })
-export class ConverterService {
-  constructor(protected injector: Injector) {}
+export class ConverterService implements OnDestroy {
+  protected subscriptions = new Subscription();
+
+  constructor(
+    protected injector: Injector,
+    protected unifiedInjector: UnifiedInjector
```

Comment:
Since this is required param and it's breaking change anyway, what about dropping obsolete `injector` constructor dependency?

...& create migration doc/schematics ticket for that